packages/ruby/README.md

```diff
@@ -11,20 +11,37 @@ Track your API metrics within ReadMe.
 `Readme::Metrics` is a Rack middleware and is compatible with all Rack-based
 apps, including Rails.
 
+When configuring the middleware, you must provide a block to tell the
+middleware how to get values for the current user. These may be values taken
+from the environment, or you may hardcode them.
+
 ### Rails
 
 ```ruby
 # application.rb
 require "readme/metrics"
 
-config.middleware.use Readme::Metrics, "YOUR_API_KEY"
+config.middleware.use Readme::Metrics, "YOUR_API_KEY"do |env|
+  {
+    id: env['CURRENT_USER'].id,
+    label: env['CURRENT_USER'].full_name,
+    email: env['CURRENT_USER'].email
+  }
```

Comment:
Thoughts on using a real gem as an example here? Say devise?

Reply:
Good idea. I'll update this